src/styles/blocks/nav-list.scss

```diff
@@ -0,0 +1,39 @@
+.nav {
+  // hide navbar for properly view for catalog with grid
+  @media (max-width: 800px) {
+    display: none;
+  }
```

Comment:
Good try, but on smaller screens, you are removing navigation completely - that is not perfect
You have either to add some navigation for smaller screens (that is out of the scope here) or just think of something else